packages/shared/src/components/auth/AuthDefault.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { ReactElement, ReactNode, useState } from 'react';
+import { CloseModalFunc } from '../modals/common';
+import AuthModalFooter from './AuthModalFooter';
+import AuthModalHeader from './AuthModalHeader';
+import AuthModalHeading from './AuthModalHeading';
+import { ColumnContainer, providers } from './common';
+import EmailSignupForm from './EmailSignupForm';
+import LoginForm from './LoginForm';
+import OrDivider from './OrDivider';
+import ProviderButton from './ProviderButton';
+
+interface AuthDefaultProps {
+  children?: ReactNode;
+  onClose?: CloseModalFunc;
+  onSignup?: (email: string) => unknown;
+  onProviderClick?: (provider: string) => unknown;
+  onForgotPassword?: () => unknown;
+  isV2?: boolean;
+}
+
+const AuthDefault = ({
```

Comment:
What exactly is AuthDefault?
It almost looks like a modal, but not really sure it is.

Reply:
Not the modal entirely but the screenshot below. The reason is, that it is also used on the login page where it is not part of any modal, but the exact same box.

![image](https://user-images.githubusercontent.com/13744167/180245014-6e79e826-a73d-4e6d-8976-eb2afbf674d8.png)

Reply:
Oh ok, then it makes sense :)